schema/event_mapping_entity.json

```diff
@@ -8,21 +8,27 @@
     "code": {
       "description": "Human readable code for the entity, should be unique in the document scope",
       "type": "string",
+      "pattern": "^[-_A-Za-z0-9]+$",
       "examples": ["fcu-8_1"]
     },
     "type": {
       "description": "DBO namespace and entity type",
       "type": "string",
+      "pattern": "^([_A-Z]+/)?[_A-Z]+$",
       "examples": ["HVAC/FCU_DFSS_DSP_CHWZC"]
     },
     "cloud_device_id": {
       "description": "Opaque identifier for the entity",
       "type": "string",
+      "pattern": "^[0-9]+$",
       "examples": ["3607425"]
     },
     "connections": {
       "description": "Entity connections, keyed by guid",
       "type": "object",
+      "patternProperties": {
+        "^[-0-9a-z]+$": { "enum": ["CONTAINS"] }
```

Comment:
the github threading is getting confused so I want to call this change out explicitly here, and I believe this is a link back to the previous thread: https://github.com/faucetsdn/udmi/pull/496/commits/8db9c3a40d6022b6ef1f34928e24886b5510aa48#r1040125597